### PR TITLE
Home works

### PR DESCRIPTION
**Pull request by SYatsun**

### CODE REVIEW

**Comment by SYatsun** [comment]
image
image
image

File: HW2/form.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    </style>
+</head>
+<body>
+	<h1> Profile edit page </h1>
+	<form action="formdata.php" method="post" name="form-name">
+	    <fieldset>
+	        <legend> Basic info</legend>
+	        <p>
+	        	<label for="id"> ID </label>
+	        	<input type="text" name="id" disabled value="725" size="50">
```
Comment (spolischook) [inline HW2/form.html:22]: ```readonly``` or ```disabled``` attribute should be applied

File: HW2/form.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+	        </p>
+	        <p>
+	        	<label for="firstname"> First Name </label>
+	        	<input type="text" name="firstname" maxlength="25" size="50">
```
Comment (spolischook) [inline HW2/form.html:26]: id attribute is missed

File: HW2/form.html [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+	        	<select name="yourCountry">
+	        		<option disabled> Choose Your Country</option>
+		            <option value="UK"> Ukraine</option>
+		            <option value="USA"> USA</option>
+		            <option value="THA"> Thailand</option>
+		            <option value="SPA"> Spain</option>
+	       		</select>
+	        </p>
+	        <p>
+	        	Skills<br>
```
Comment (spolischook) [inline HW2/form.html:96]: duplicated and meaningless values for checkboxes

File: HW2/form.html [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+	        <p> Development Experience <br>
+	            <label><input type="radio" name="experience" value="Learning"> Learning to Program </label><br>
+	            <label><input type="radio" name="experience" value="Junior" checked> Junior Developer </label><br>
+	            <label><input type="radio" name="experience" value="Mid-Level"> Mid-Level Developer </label><br>
+	            <label><input type="radio" name="experience" value="Senior"> Senior Developer </label><br>
+	        <p>
+	        	Country
+	        	<select name="yourCountry">
+	        		<option disabled> Choose Your Country</option>
+		            <option value="UK"> Ukraine</option>
```
Comment (spolischook) [inline HW2/form.html:89]: Ukraine is UA not UK

File: HW2/form.html [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+	        </p>
+	        </p>
+	        <p> Development Experience <br>
+	            <label><input type="radio" name="experience" value="Learning"> Learning to Program </label><br>
+	            <label><input type="radio" name="experience" value="Junior" checked> Junior Developer </label><br>
+	            <label><input type="radio" name="experience" value="Mid-Level"> Mid-Level Developer </label><br>
+	            <label><input type="radio" name="experience" value="Senior"> Senior Developer </label><br>
+	        <p>
+	        	Country
+	        	<select name="yourCountry">
```
Comment (spolischook) [inline HW2/form.html:87]: Preferred choices is missed. See wire-frame

**Review by spolischook — CHANGES_REQUESTED** [review]
I've suggest use ```div``` as wrapper for form fields instead of ```p```, however Yana suggest use ```li``` and I'm agree, because it's semantically right

```id``` attribute is required for field in case if label has ```for``` attribute https://www.w3schools.com/tags/att_for.asp

I've suggest use ```div``` as wrapper for form fields instead of ```p```, however Yana suggest use ```li``` and I'm agree, because it's semantically right

**Pull request closed without merging**